elastic/kibana
# [Alerting] Edit alert should show and update all actions with deleted connectors

Resolves https://github.com/elastic/kibana/issues/86695

## Summary

Fixed bug where not all actions using a deleted connector were showing up on the "Edit Alert" flyout. Updated behavior when creating new connector from the broken connector callout so that all actions using the broken connector are updated to use the new connector instead of just the action you clicked on.

To verify:
1. Create a connector.
2. Create an alert with multiple actions that use the connector from (1)
3. Delete the connector from (1)
4. Edit the alert from (2)
5. Verify that multiple action sections show up, one for each of the actions that use the deleted connector
6. Verify that the Save button is enabled if you remove all the actions for the deleted connector
7. Verify that if you click "Create new connector" from one of the actions for the deleted connector, the new connector is applied to all the actions that used the deleted connector and the Save button is enabled.

### Checklist

Delete any items that are not applicable to this PR.
